tools/bin/go_core_tests

```diff
@@ -8,10 +8,9 @@ grep "\-\-\- FAIL" output.txt
 grep "FAIL	" output.txt
 grep "driver: bad connection" output.txt
 echo "Exit code: $EXITCODE"
-if [[ $EXITCODE == 1 ]]; then
+if [[ $EXITCODE != 0 ]]; then
```

Comment:
Why this change?

Reply:
`go test` can return error codes other than `1`. I had a test run where the entire suite crashed, but the CI considered it green. Only `0` needs to be considered a success